5-logging/bookshelf/__init__.py

```diff
@@ -34,9 +36,11 @@ def create_app(config, debug=False, testing=False, config_overrides=None):
         app.config.update(config_overrides)
 
     # [START setup_logging]
-    # Configure logging
     if not app.testing:
-        logging.basicConfig(level=logging.INFO)
+        client = google.cloud.logging.Client(app.config['PROJECT_ID'])
+        handler = CloudLoggingHandler(client)
+        logging.getLogger().setLevel(logging.INFO)
+        setup_logging(handler)
```

Comment:
Attaches the handler to the root logger, so just plain `logging.info` shows up in the UI rather than using a handler.

The actual method does some bookkeeping stuff, primarily it makes sure it excludes the code path of the logging handler itself to avoid infinite recursion.